src/start_server.ts

```diff
@@ -113,6 +116,22 @@ export function getApp(options: ServerOptions) {
   return app;
 }
 
+/**
+ * Open the given web page URL. If no browser keyword is provided, `opn` will use
+ * the user's default browser.
+ */
+function openWebPage(url: string, withBrowser?: string) {
+  let openOptions = {
+    app: withBrowser
+  };
+  opn(url, openOptions, (err) => {
+    if (err) {
+      console.log(`Error launching "${openOptions.app || 'default web browser'}".`);
```

Comment:
Yea I torn on this too, and don't really like how either one looks code-wise. 

As the user I'd rather be told immediately that my command wouldn't run as expected and why so that I can fix it vs. swallowing or logging it. Although if the command is wrapped/abstracted over by another tool and the user doesn't call this themselves, then if something breaks they can't fix it. That alone might be a good reason to log & continue.